source/includes/inkling-reference/_concepts.html.md

```diff
@@ -56,38 +54,35 @@ outputTarget ::=
+are listed after the `follows` keyword and output sources are listed after the
+`feeds` keyword. 
 
-The `is` clause characterizes output. The `is classifier` form specifies that the output is an enum. 
+Input sources can be other concepts or the `input` stream.  The `input` stream
+is the original input to the system. It flows into the system from outside the
+BRAIN. Each reference to the `input` stream must have a schema, which can be
+anonymous. (An [anonymous schema][1] is a list of schema fields in place of a schema name).
 
-------> from web
+Output targets can also be other concepts or the `output` stream. The `output` stream refers to the output of the BRAIN. 
+The `output` stream is never referenced with a schema.
 
-* The concept must be named after the `concept` keyword.
-* The `is` keyword specifies the kind of prediction the trained concept will produce. For example, a concept can specify `is classifier`. This means that the trained concept will categorize its input. Email, for example, can be classified as spam or not spam. Another option with this keyword is `estimator`.
-* The concept must declare an output schema after `predicts`. The output schema describes the data produced by the trained concept. For example if this concept classifies email into spam and not spam, the output schema for the concept would be a Bool. The output schema can be a named schema, where the name refers to a full schema definition elsewhere, or it can be anonymous, which is a parenthesized list of name, type pairs. See the section on schema declarations for more information.
-* A trained concept gets input from streams or (if multiple concepts are used) from another concept. The `input` keyword refers to the stream that is the original input to the system. All data flowing through the system has a schema associated with it. In some cases this is calculated rather than explicit.
-* If the `input` keyword appears in the `follows` list, it means that the input stream flowing into this concept comes from outside the BRAIN. The `input` keyword must always be accompanied by a schema (named or anonymous) because the data stream originates outside the Brain; if no schema was present, data types and formats being input would be unknown.
-* The `feeds` list is a list of concepts and streams (including the predefined output stream) for which the output of this concept is a source.
-* The `input` keyword cannot not appear in the feeds list and the `output` keyword cannot appear in the follows list.
-* The concept statement is terminated by the `end` keyword.
+The concept output is described in the `predicts` clause. The schema after the
+`predicts` keyword describes the data produced by the trained concept.  (This
+schema is required but it can be [anonymous][1].)
+For example, if this concept classifies email into spam and not spam, the output schema for the concept would be a Bool. 
 
-### Discussion
```

Comment:
This was part of the outline that we came up with before. Do you not want a Discussion section for any of the top level headings anymore or just for this one?